pages/candidates.tsx

```diff
@@ -1,14 +1,18 @@
 ﻿import React from "react";
 import {NextPage} from "next";
-import textStyle from "../pageStyles/text-classes.module.scss"
+import textStyle from "../pageStyles/text-classes.module.scss";
+import scss from "../pageStyles/candidates.module.scss";
 import Layout from "../components/Layout/layout";
 import {CandidateList} from "../components/CandidateList/CandidateList";
+import Link from "next/link";
 
 const Candidates: NextPage = () => {
 
     return (
         <Layout>
             <h1 className={textStyle.pageHeader}>Candidate Manager</h1>
+            <button className={scss.desktopButton}><Link href={"/create"}><p className={scss.link}>+ Create Candidate</p></Link></button>
+            <button className={scss.mobileButton}><Link href={"/create"}><p className={scss.link}>+</p></Link></button>
```

Comment:
Couple of things

1) this 'button' is really just a link - no need to wrap it in a 'button' - just style the link to look how you like.
2) in Next, `Link` elements are meant to just be wrappers around a normal `a` tag. That way, without JS they work just like normal, and when you add JS then you get pre-fetching and other nice things. - https://nextjs.org/docs/api-reference/next/link

Could you get rid of the button, and place an `a` tag inside the link?

Reply:
Yep replaced the button